Modelica/Fluid/Examples/Tanks.mo

```diff
@@ -226,7 +226,7 @@ simulation accuracy could be increased in order to avoid errors.
       crossArea=1,
       level_start=1,
       portsData={Modelica.Fluid.Vessels.BaseClasses.VesselPortsData(diameter=
-          0.1)},
+          0.1, height=0.001)},
```

Comment:
Should it be a constant?

Reply:
Yes. I mean, when you parameterize the tank model, you pass it a VesselPortsData which is built on the fly with the parameter values that you need. The height of the port is not different from its diameter. Do I miss something?

Reply:
I believe the idea was to have a common parameter (or constant) for the bottom-outlet and the initial tank-level.
To me that does not seem necessary, and I prefer to keep those two as different parameters even if they are related.

Reply:
> I believe the idea was to have a common parameter (or constant) for the bottom-outlet and the initial tank-level.

No, that's never been the case. One is the height of the bottom outlet _of the first tank_, the other is the initial level of the second tank. They are not related by any means. They are just meant to be some small values, since a zero value is numerically problematic and hence forbidden.

> To me that does not seem necessary, and I prefer to keep those two as different parameters even if they are related.

Just to be clear, they are not.

Reply:
> Yes, two constants are even clearer. In the new explanation text, 1 mm is mentioned multiple times. It would be helpful to have a referecence from text to the actual parameter.

@beutlich there is really no need to define _symbolic_ constants for this. _Literal_ constants are good enough. We use them all the time to parameterize models.